examples/graphql-subscriptions/graphql_subscriptions.bal

```diff
@@ -1,22 +1,32 @@
 import ballerina/graphql;
+import ballerina/random;
+import ballerina/lang.runtime;
 
 service /graphql on new graphql:Listener(9090) {
-    // Defines a `string` array in the service.
-    private string[] names;
 
-    function init() {
-        // Initialize the array.
-        self.names = ["Walter White", "Jesse Pinkman", "Skyler White"];
-    }
-
-    resource function get names() returns string[] {
-        return self.names;
+    resource function get greeting() returns string {
+        return "Hello, World!";
     }
 
     // A resource method with the `subscribe` accessor represents a field in the root
     // `Subscription` operation. It must always return a stream. Since the stream is of type
     // `string`, the resulting field in the generated GraphQL schema will be of type `String!`.
-    resource function subscribe names() returns stream<string> {
-        return self.names.toStream();
+    resource function subscribe names() returns stream<string, error?> {
+        NameGenerator nameGenerator = new;
+        stream<string, error?> names = new (nameGenerator);
+        return names;
+    }
+}
+
+// Defines a stream generator class that generates random names.
```

Comment:
Should we include that it returns with a delay?

Reply:
And maybe say that this can be used to create a stream of values? :thinking: Since users may not be familiar with this.